recipes/xgrads/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "xgrads" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 927c84c615123ee4ddc98a499da70f06341092e911846735c6f822d6970ef621
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
```

Comment:
It seems like this package could probably be noarch. Can you please check?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds
Easier way to generate the package: https://github.com/conda/grayskull